README.md

```diff
@@ -94,7 +94,7 @@ Configuration Reference
 
  * `url` (required): The URL to your Elasticsearch bulk API endpoint
  * `index` (required): Name if the index to publish to (populated using PatternLayout just like individual properties - see below)
- * `type` (optional): Elasticsearch `_type` field for records
+ * `type` (required): Elasticsearch `_type` field for records
```

Comment:
Can't type be encoded in the URL (`{index}/{type}/_bulk`) which would make this field optional?